internal/graph/lookup.go

```diff
@@ -112,7 +111,7 @@ func (cl *ConcurrentLookup) LookupViaReachability(ctx context.Context, req Valid
 		Metadata:   req.Metadata,
 	}, stream)
 	if err != nil {
-		resp := lookupResultError(NewErrInvalidArgument(fmt.Errorf("error in reachablility: %w", err)), emptyMetadata)
+		resp := lookupResultError(err, emptyMetadata)
```

Comment:
this was turning every error received by reachable resources into an invalid argument, and prevented downstream code to provide a more specific error code. It also made the assumption everything fell in that category, whereas other error handling in this method did not make such assumption. Therefore it seemed uncontroversial to remove this.